### ISSUE TITLE
Add scripts for common tasks?

### ISSUE THREAD

**Issue by mrlubos** [lllychen/jekyll-firewall#4]
The project currently misses `scripts` inside `package.json`. It would make life easier if we could use package manager to run the most frequently used scripts, e.g. `serve` or `build`.

**Comment by lilykonings** [comment]
I want to include only what is pertinent so users don't have to spend any time re-writing or deleting extra stuff. [This commit](https://github.com/lllychen/jekyll-firewall/commit/059bc15e71d4003293288897e65164aaf95d7009) separated out firewall task to give room for custom tasks

**Comment by mrlubos** [comment]
Good idea with the separation. I guess it comes down to what's the primary use case for this library. Should the user implement it into her project (integrate), or build on top of it (boilerplate)? If it's the former, this project could be narrowed down pretty much to the `gulpfile.js` or rather those few crucial tasks. If it's the latter, I'd compare it to [`create-react-app`](https://github.com/facebook/create-react-app) and argue to include some scripts. 🙌
